sklearn/datasets/base.py

```diff
@@ -512,23 +518,31 @@ def load_diabetes(return_X_y=False):
     -------
     data : Bunch
         Dictionary-like object, the interesting attributes are:
-        'data', the data to learn and 'target', the regression target for each
-        sample.
+        'data', the data to learn, 'target', the regression target for each
+        sample, 'data_filename' (added in version 0.19), the physical location
```

Comment:
I think the point of `versionadded` and `versionchanged` is that they can appear out of the way of the main description of the data. For most users, the structure of the data is what's relevant, not its history.